source/store.go

```diff
@@ -281,19 +281,20 @@ func NewIstioClient(kubeConfig string) (*istiocontroller.Client, error) {
 		}
 	}
 
-	client, err := istiocontroller.NewClient(
-		kubeConfig,
-		"",
-		istiomodel.ConfigDescriptor{istiomodel.Gateway},
-		"",
-	)
-	if err != nil {
-		return nil, err
-	}
+	return nil, errors.New("Istio needs to be update to support latest version")
+	//client, err := istiocontroller.NewClient(
+	//	kubeConfig,
+	//	"",
+	//	istiomodel.ConfigDescriptor{istiomodel.Gateway},
+	//	"",
+	//)
+	//if err != nil {
+	//	return nil, err
+	//}
```

Comment:
Was this commented out during debugging?

Reply:
Not quite, istio needs to be updated to support its new API. That is why I had to comment out this portion.